public/wp-content/themes/.gitignore

```diff
@@ -0,0 +1,4 @@
+# ignore all themes except newsup-based themes and .gitignore
+*
+!.gitignore
+!public/wp-content/themes/newsup*
```

Comment:
You need to add the all-in-one-wp-migration to the .gitignore because about 170+ of these changed files are from that.

Reply:
i added the entire plugins folder to my gitignore

does that solve the problem?

Reply:
> 1. Use sidebar to display review links.
> 2. Use "You Missed" to display anime related to current by genre (any order).

1. i have no clue how to modify the widgets to take in custom types
2. i loaded in anime instead of posts, but i dont know how to pull the genre of the page since its in the footer. im thinking this is very non-essential, and if you want a related anime section, i could do that in another card within single-anime.php. leme know what you think :)

Reply:
> i added the entire plugins folder to my gitignore
> 
> does that solve the problem?

Yup

> > 1. Use sidebar to display review links.
> > 2. Use "You Missed" to display anime related to current by genre (any order).
> 
> 1. i have no clue how to modify the widgets to take in custom types
> 2. i loaded in anime instead of posts, but i dont know how to pull the genre of the page since its in the footer. im thinking this is very non-essential, and if you want a related anime section, i could do that in another card within single-anime.php. leme know what you think :)

Yeah I'm thinking we could just skip on this part unless anyone wants to tinker around with it just for the experience.

Reply:
agreed.